NOTICES

```diff
@@ -5,7 +5,7 @@ This file details additional third party software license agreements
 and third party notices and information that are required 
 to be reproduced for the following programs:
    
-Operator for IBM Block Storage CSI Driver version 1.3.0
+Operator for IBM Block Storage CSI Driver version 1.4.0

```

Comment:
does this affect the cert checks? because if not, it should be done in a separate licensing ticket

Reply:
Not sure, most probably the answer is no, I will revert the changes for this file (thanks)

Reply:
done